source/features/link-to-requested-change-comment.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function init(): void {
+	observe('.review-item .dropdown-item[href^="#pullrequestreview-"]', {
```

Comment:
We also need to add a `.rgh` class to mark the element as "processed". Refer to the other features using selector-observer